docs/vsphere_ci/scripts/configure-vm-tools.ps1

```diff
@@ -6,7 +6,7 @@
 # download configuration template
 Invoke-WebRequest -O tools.conf https://raw.githubusercontent.com/vmware/open-vm-tools/master/open-vm-tools/tools.conf
 # exclude network interface
```

Comment:
Your commit message says what the commit does but does not call out what issue it is fixing. Please mention that explicitly. Just saying `where all interfaces are included and properly reported by VMTools in the vCenter.` does not make it obvious. 
I also suggest mentioning that in this comment.